src/components/PriceBar/BudgetBoard.tsx

```diff
@@ -1,22 +1,25 @@
-import { Button, Center, Text } from "@chakra-ui/react";
+import { Button, Center, Text, useToast } from "@chakra-ui/react";
 import { useStoreSelector } from "@/store/hooks";
 import { useAddToCartMutation } from "@/redux/api/cartApi";
 import { ICartItem } from "@/interfaces/cartItem";
 import { saveDesignMapping } from "@/utils/designMapping";
 import { IDesign, ITileColor } from "@/interfaces/design";
 import { useDispatch } from "react-redux";
 import { switchLoginModal } from "@/store/reducer/buttonToggleSlice";
+import { upLoadScreenshot } from "@/utils/uploadImage";
 
 interface IBudgetBoardprops {
   useTotalPrice: string;
 }
 
 const BudgetBoard = ({ useTotalPrice }: IBudgetBoardprops) => {
   const dispatch = useDispatch();
+  const toast = useToast();
 
   const tileBlocks = useStoreSelector((state) => state.priceBar.blocks);
   const court = useStoreSelector((state) => state.courtSpecData).activeCourt;
   const tileData = useStoreSelector((state) => state.tile.present.court);
+  const screenshot = useStoreSelector((state) => state.canvasControl.screenshot);
```

Comment:
It seems like this screenshot is a string of URL addresses? If so then screenshot is not an appropriate name to identify the real value of it. The logic here is a little confuse me.

Reply:
It's named courtDataUrl now